shell/browser/electron_browser_main_parts.cc

```diff
@@ -539,6 +541,16 @@ void ElectronBrowserMainParts::PreMainMessageLoopStartCommon() {
   RegisterURLHandler();
 #endif
   media::SetLocalizedStringProvider(MediaStringProvider);
+
+#if defined(OS_WIN)
+  if (electron::fuses::IsCookieEncryptionEnabled()) {
+    auto* local_state = g_browser_process->local_state();
+    DCHECK(local_state);
+
+    bool os_crypt_init = OSCrypt::Init(local_state);
+    DCHECK(os_crypt_init);
```

Comment:
I feel like this should maybe be higher than a `DCHECK`?

Reply:
c.f. https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/chrome_browser_main_win.cc;l=583-584?q=OSCrypt::Init&ss=chromium

Reply:
hm, ok.